mysql-ballerina/client.bal

```diff
@@ -161,10 +161,17 @@ public type Options record {|
     string serverTimezone?;
 |};
 
-# Possible options for SSL Mode.
+# Establish an encrypted connection if the server supports encrypted connections, falling back to an unencrypted
+# connection if an encrypted connection cannot be established.
 public const SSL_PREFERRED = "PREFERRED";
+# Establish an encrypted connection if the server supports encrypted connections. The connection attempt fails if
+# an encrypted connection cannot be established.
 public const SSL_REQUIRED = "REQUIRED";
+# Like SSL_REQUIRED, but additionally verify the server Certificate Authority (CA) certificate against the configured
```

Comment:
instead of using `Like SSL_REQUIRED `, shall we explain it. it's okay to duplicate

Reply:
explanations added in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/157/commits/98bb69547c284bf2186bc2a14cbb622cd2fc34ad